pycqed/measurement/pulse_sequences/single_qubit_tek_seq_elts.py

```diff
@@ -105,57 +109,74 @@ def rabi_seq_active_reset(amps, qb_name, operation_dict, cal_points,
     # add calibration segments
     seq.extend(cal_points.create_segments(operation_dict, **prep_params))
 
+    # reuse sequencer memory by repeating readout pattern
+    seq.repeat_ro(f"RO {qb_name}", operation_dict)
+
     log.debug(seq)
     if upload:
        ps.Pulsar.get_instance().program_awgs(seq)
 
     return seq, np.arange(seq.n_acq_elements())
 
 
-def T1_seq(times, pulse_pars, RO_pars,
-           cal_points=True, upload=True, return_seq=False):
+def t1_active_reset(times, qb_name, operation_dict, cal_points,
```

Comment:
`t1_active_reset` -> `t1_seq`?

Reply:
This naming is used to differentiate sequence which are in the new framework and handle both ge and ef pulses. Once we have removed all the old ones (which we should, I already did most of them but Im not sure in the remaining ones which ones could still be used), then I will rename all of the sequences :)

Reply:
Ok, agreed that the different naming convention is a good clue for the framework version it uses.

Reply:
I guess we could `class T1(swf.Hard_Sweep)` then, since it references a nonexistant `sqs.T1_seq`. Also other AWG sweep functions that reference sequence functions that don't exist.

Reply:
I will remove the `class T1`